components/mintKamon/GenerateImageForm.tsx

```diff
@@ -27,7 +27,7 @@ const GenerateImageForm: React.FC<Prop> = ({
   onSetTokenImageURI,
   address
 }) => {
-  const ipfsApiEndpoint = 'https://api.staging.sakazuki.xyz/henkaku/ipfs'
+  const ipfsApiEndpoint = process.env.NEXT_PUBLIC_IPFS_API_URI + '/ipfs'
```

Comment:
/ipfs should be also in env

Reply:
Until there's another endpoint we want to hit?

No problem with that, though! Will do.